File: src/com/patterns/creational/builder/java/model/Phone.java
package com.patterns.creational.builder.java.model;

public class Phone {

    private String os;
    private int ram;
    private double screenSize;

    public Phone(String os, int ram, double screenSize) {
        this.os = os;
        this.ram = ram;
        this.screenSize = screenSize;
    }

    @Override
    public String toString() {
        return "Phone{" +
                "os='" + os + '\'' +
                ", ram=" + ram +
                ", screenSize=" + screenSize +
                '}';
    }
}

File: src/com/patterns/creational/builder/korlin/model/Phone.kt
package com.patterns.creational.builder.korlin.model

class Phone(
    var os: String? = null,
    var processor: String? = null,
    var screenSize: Double? = null,
    var battery: Int? = null,
    var camera: Int? = null
) {
    override fun toString(): String {
        return "Phone(os=$os, processor=$processor, screenSize=$screenSize, battery=$battery, camera=$camera)"
    }
}